### ISSUE TITLE
Not so Native ... (NDK)

### ISSUE THREAD

**Issue by rezamarzban** [logits-systems/Cam2Ndk#3]
Hi, I can research and develope some codes in C++, Python, PHP, Java script programing languages and so on easily because I have about 20 years experiment in IT. Recently I decide to research in android codes (Kotlin and Java), But after a week research in I am confused.
The android project which you created here in Native C++ is not so Native, It's confusing ... .
My recent android project in Kotlin is more native!

https://github.com/marzban2030/CameraX-tflite

**Comment by rezamarzban** [comment]
I deleted and recreated above repository with different name, Instead look at below repository which is more native than NDK!

https://github.com/marzban2030/TFLiteObjectDetection

This repository contains most simple and accurate Android TFlite project with fewest codes and components.

Finally, After a month I can develope Kotlin and Java codes easily!
At past month I was confusing ...
